UPGRADE.md

```diff
@@ -2798,3 +2798,96 @@ import { BooleanInput } from 'react-admin';
 +   onChange={e => { console.log(e.target.checked); }}
 />
 ```
+
+## `<ReferenceInput>` and `<ReferenceArrayInput>` Now Provide A `ChoicesContext`
+
+`<ReferenceInput>` and `<ReferenceArrayInput>` used to inject props to their children. They now provide a `ChoicesContext` with support for sorting, paginating and filtering through methods and properties that match those of the `ListContext`. All their methods now have the same signature. As a result, several changes were made to both components:
+
+### `<ReferenceInput>` No Longer Accepts The `filterToQuery` Prop
```

Comment:
`ReferenceInput` still has `filterToQuery` prop referenced in the code

Reply:
Fixed

Reply:
btw, adding comments where the issue is would allow github to automatically add the "outdated" flag on the PR comment ;)